test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,74 @@
+const agent = require('superagent');
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('First Api Tests', () => {
+});
+
+it('Consume GET Service', async () => {
```

Comment:
Los it van dentro del describe porque el describe define un contexto para las pruebas. Las pruebas son los it 👎